tensorflow/python/ops/linalg_grad.py

```diff
@@ -301,7 +301,7 @@ def _SvdGrad(op, grad_s, grad_u, grad_v):
         "inner matrix shape.")
 
   if not op.get_attr("compute_uv"):
-    s, u, v = linalg_ops.svd(a, compute_uv=True, full_matrices=True)
+    s, u, v = linalg_ops.svd(a, compute_uv=True)
```

Comment:
Should we not compute the full matrices if op.get_attr("full_matrices") == True? See line 375..
BTW, I would add:

full_matrices = op.get_attr("full_matrices") 
comput_uv = op.get_attr("compute_uv")

at the top of the function to improve readability.

Reply:
> Should we not compute the full matrices if op.get_attr("full_matrices") == True? See line 375.

There's no need -- if `full_matrices` is true, then the function returns early and never reaches line 375: https://github.com/vnavkal/tensorflow/blob/66a8c03a8eaf99de5eca41e1f7a7bf9d0209539d/tensorflow/python/ops/linalg_grad.py#L321-L329.  The change I suggested in my comment above should make this less confusing; I'll implement it.

> BTW, I would add:
> ```
> full_matrices = op.get_attr("full_matrices") 
> compute_uv = op.get_attr("compute_uv")
> ```
> at the top of the function to improve readability.

sounds good, I'll do this.